docs/reference.md

```diff
@@ -258,4 +266,54 @@ applied if the marked test method fails or errors.
 Log a summary of the status of the model. This is automatically called before the model
 is cleaned up.
 
+#### `async def track_model(self, alias: str, model_name: Optional[str] = None, cloud_name: Optional[str] = None, create: bool = True, **kwargs,) -> Model`
+
+Indicate to `ops_test` to track a new model which is automatically created in juju or an existing juju model referenced by model_name. 
+This allows support for `ops_test` to track multiple models on various clouds by a unique alias name.
+
+#### `async def forget_model(self, alias: str, timeout: Optional[Union[float, int]] = None)`
+
+Indicate to `ops_test` to forget an existing model.
+* If this model was intended to be kept, the model will be forgotten by the test structure immediately
+* If not, the model will attempt to be destroyed and removed from the controller.  
```

Comment:
I didn't understand this.  "if this model was intended to be kept" - is that something like if `--keep-models` is set?

Reply:
there are two ways a model is "kept":
1) `--keep-models` is passed on the command line to tox
2) `--model=<a model name>`,  ie `ops_test` doesn't create the model

Reply:
ok gotcha.  Maybe add a `ModelState.keep == True` or some other appropriate statement.  But maybe this is just my ignorance about how pytest-operator works.  Whatever you think is best